.github/workflows/port-issue.yaml

```diff
@@ -12,9 +13,23 @@ jobs:
     runs-on: ubuntu-latest
     if: contains(github.event.comment.body, '/backport') || contains(github.event.comment.body, '/forwardport')
     steps:
+      - name: Read secrets
+        uses: rancher-eio/read-vault-secrets@main
+        with:
+          secrets: |
+            secret/data/github/repo/rancher/${{ github.repository }}/github/app-credentials appId | APPID;
+            secret/data/github/repo/rancher/${{ github.repository }}/github/app-credentials privateKey | PRIVATEKEY
+
+      - name: Generate Token
+        id: generate-token
+        uses: actions/create-github-app-token@v1
```

Comment:
.github/workflows/pr-gh-project.yaml uses v2, not sure that's the latest but it should be at least that